NGitLab.Mock.Tests/MilestonesMockTests.cs

```diff
@@ -99,5 +99,43 @@ public void Test_milestones_can_be_closed_and_activated_from_project()
             Assert.AreEqual(1, activeMilestones.Length, "Active milestones count is invalid");
             Assert.AreEqual(0, closedMilestones.Length, "Closed milestones count is invalid");
         }
+
+        [Test]
+        public void Test_projects_merge_request_can_be_found_from_milestone()
+        {
+            using var server = new GitLabConfig()
+                .WithUser("user1", isDefault: true)
+                .WithProject("Test", id: 1, addDefaultUserAsMaintainer: true, configure: project => project
+                    .WithMilestone("Milestone 1", id: 1)
+                    .WithMergeRequest("branch-01", title: "Merge request 1", milestone: "Milestone 1")
+                    .WithMergeRequest("branch-02", title: "Merge request 2", milestone: "Milestone 1")
+                    .WithMergeRequest("branch-03", title: "Merge request 3", milestone: "Milestone 2"))
+                .BuildServer();
+
+            var client = server.CreateClient();
+            var mergeRequests = client.GetMilestone(1).GetMergeRequests(1).ToArray();
```

Comment:
For clarity as both Ids are '1': can you put projectId and milestoneId in variables?

As such, will be less confusing:

```csharp
client.GetMilestone(1).GetMergeRequests(1).ToArray();

// Will be clearer
client.GetMilestone(projectId).GetMergeRequests(milestoneId).ToArray();
```